Controller/Activation/ActivationController.php

```diff
@@ -110,21 +111,18 @@ public function activateAction(string $token): Response
             return new RedirectResponse(
                 $this->router->generate('fsi_admin_activation_change_password', ['token' => $token])
             );
-        } else {
-            $this->eventDispatcher->dispatch(
-                AdminSecurityEvents::ACTIVATION,
-                new ActivationEvent($user)
-            );
-
-            return $this->addFlashAndRedirect('success', 'admin.activation.message.success');
```

Comment:
Well if you really want to refactor this, you could leave the `if/else` but assign the response to a variable, so there's only one `return` statement.